### PR TITLE
[198] - Printing Product Price with JavaScript - Fullstack

### PR DESCRIPTION
**Pull request by misturaeneeolah**

### CODE REVIEW

File: submissions/masturohAl-Ameen/javaScript/printingProductPrice.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var productList = [
+    {
+        productName: "Shorts",
+        imageURL: "images/shorts.jpg",
+        productPrice: "#20,000",
+    },
+
```
Comment (TundeOladejo) [inline submissions/masturohAl-Ameen/javaScript/printingProductPrice.js:7]: Remove space here

File: submissions/masturohAl-Ameen/javaScript/printingProductPrice.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        productName: "Nightwear",
+        imageURL: "images/nightwear.jpg",
+        productPrice: "#10,000",
+    },
+
```
Comment (TundeOladejo) [inline submissions/masturohAl-Ameen/javaScript/printingProductPrice.js:13]: Here too

**Review by TundeOladejo — COMMENTED** [review]
It seems you forgot to add the task id to the title and remember to specify the track you are working on also.

File: submissions/masturohAl-Ameen/javaScript/printingProductPrice.js (hunk 2 above)
Comment (TundeOladejo) [inline submissions/masturohAl-Ameen/javaScript/printingProductPrice.js:13]: Remove space here too

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Comment by misturaeneeolah** [comment]
Thank you very much. I will make the necessary corrections now.

**Pull request closed without merging**